docs/refcard.adoc

```diff
@@ -42,35 +42,35 @@ a|`surefire`, `failsafe`
 |===
 |Property | Description | Default | Applicable Strategies
 
-a|`scm.range.tail`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="PREVIOUS_COMMIT"]`
 |Set first commit id for inspecting changes
-a|`HEAD~0`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="HEAD"]~const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="DEFAULT_LAST_COMMITS"]`
 a|`new`, `changed`, `affected`
 
-a|`scm.range.head`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]`
 |Set last commit id for inspecting changes
-a|`HEAD`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="HEAD]`
 a|`new`, `changed`, `affected`
 
-a|`scm.last.changes`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]`
 |Set the number of commits from `HEAD` that you want to inspect
-a|`0`
+a|`const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="DEFAULT_LAST_COMMITS"]`
 a|`new`, `changed`, `affected`
 
-a|`smart.testing.affected.inclusions`
+a|`const:strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedRunnerProperties.java[name="SMART_TESTING_AFFECTED_INCLUSIONS"]`
 |Set classes to be included for scanning
 a|
-a|`affected`
```

Comment:
Here and in next three cases, you removed the information about which strategy is the variable applicable for.

Reply:
ouch